common/config/rush/pnpm-lock.yaml

```diff
@@ -6149,8 +6114,8 @@ packages:
     resolution: {integrity: sha1-EUyUlnPiqKNenTV4hSeqN7Z52is=}
     dev: false
 
-  /moment/2.29.2:
-    resolution: {integrity: sha512-UgzG4rvxYpN15jgCmVJwac49h9ly9NurikMWGPdVxm8GZD6XjkKPxDTjQQ43gtGgnV3X0cAyWDdP2Wexoquifg==}
+  /moment/2.29.1:
```

Comment:
@colawwj I noticed that `moment` got downgraded and re-introduced a security issue: https://github.com/moment/moment/security/advisories/GHSA-8hfj-j24r-96c4

could you please share how you resolved the conflict with pnpm-lock.yaml at upstream/main? I'd like to understand what caused the downgrade.  Normally we would do
```shell
git merge upstream/main
```
this gives a merge conflict for pnpm-lock.yaml, we then get pnpm-lock.yaml from upstream/main
```shell
git checkout upstream/main common/config/rush/pnpm-lock.yaml
```
then run `rush update` on top of it.